index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="Style.css">
+</head>
+<body>
+    
+            <header class="navbar">
+                <nav> 
+                <div> 
+                <a class="logo" >Abdullah</a>
+         
+            <span>
+            <a class="union" href="#" ><img src="Union.png"></a> 
+            </span>
+        </nav>           
+            </div>
+        </header>
+   <section class="flex-cont">
+    
+    
+    
+        
+        <h1 class="name">
+            Hey there! I'm Abdullah 
+        </h1>
+        <h3 class="job">
+            I'm a softwar developer
+        </h3>
```

Comment:
- [ ] I also noticed you changed the color of the headline text and it also contains a typo in the `software` spelling. Kindly ensure the color matches the Figma design text color 👍🏽 

| Figma Design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/212173753-644c9e40-4f99-4f95-99c8-8985fd6953c6.png)  | ![image](https://user-images.githubusercontent.com/63473584/212173944-9b021a2c-ccb6-4b43-9c90-c12e824ec75d.png)  |